src/components/FeatureViews/Search/UpcomingSessionCards.js

```diff
@@ -0,0 +1,99 @@
+import { connect } from "react-redux";
+import React, { useState } from "react";
+import BackButton from "../../BackButton";
+import Grid from "@material-ui/core/Grid";
+import { Card, Paper, Typography, Grow } from "@material-ui/core";
+
+import Chip from "@material-ui/core/Chip";
+import CardActions from "@material-ui/core/CardActions";
+import { withRouter } from "react-router-dom";
+
+
+
+function UpcomingSessionCards(props) {
+    const [value, setValue] = useState(0);
+    const [userList, setUserList] = useState([])
+    const [viewToggle, setViewToggle] = useState(true);
+
+    const goToRoute = (route) => {
+        props.history.push(props.match.url + route);
+    }
+
+
+
+    return (
+        <Grid item xs={12} sm={3} className={"UpcomingSessionCards"} style={{ padding: "20px" }}>
+            <Card key={props.user.user_id}
+                style={{ cursor: "pointer" }}
+                onClick={(event) => {
+                    event.preventDefault();
+                    goToRoute(`/${props.user.role}/${props.user.user_id}`);
+                }}>
+                <Grid container>
+                    <Grid item sm={12}>
+                        <Typography align={"center"} variant={"h5"}> Noveber 12, 2019 | 7:00 - 9:00 </Typography>
```

Comment:
Even for testing, retrieve a course from redux and display it by that course's data - easier to plug in actual search results, and easier to test with different data.